hare/hare_shared.go

```diff
@@ -0,0 +1,42 @@
+package hare
+
+import (
+	"github.com/spacemeshos/go-spacemesh/p2p/service"
+	"hash/fnv"
+	"time"
+)
+
+const ProtoName = "HARE_PROTOCOL"
+const RoundDuration = time.Second * time.Duration(15)
+const (
+	Status   = 0 // round 0
```

Comment:
i think its better if we define this enum in either the proto file or here
doing it twice may cause issues in the future